test/test_summary_report_builder.rb

```diff
@@ -0,0 +1,20 @@
+require 'test/unit'
+require_relative '../lib/pmdtester/builders/summary_report_builder'
+require_relative '../lib/pmdtester/parsers/projects_parser'
+
+# Unit test class for PmdTester::SummaryReportBuilder
+class TestSummaryReportBuilder < Test::Unit::TestCase
+  def test_summary_report_builder
```

Comment:
We probably need here a `rake clean` call, too.

Reply:
Done.